lib/galaxy/webapps/galaxy/fast_app.py

```diff
@@ -156,20 +157,26 @@ def get_fastapi_instance() -> FastAPI:
     )
 
 
-def get_openapi_schema() -> Dict[str, Any]:
+def get_openapi_schema(version="3.1.0") -> Dict[str, Any]:
     """
     Dumps openAPI schema without starting a full app and webserver.
     """
     app = get_fastapi_instance()
     include_all_package_routers(app, "galaxy.webapps.galaxy.api")
-    return get_openapi(
+    schema_3_1_0 = get_openapi(
         title=app.title,
         version=app.version,
-        openapi_version="3.1.0",
+        openapi_version=version,
         description=app.description,
         routes=app.routes,
         license_info=app.license_info,
     )
+    if version.startswith("3.0"):
```

Comment:
There's more, `examples` in 3.0 is example in 3.1

Reply:
If it isn't linted - it doesn't exist 😆. I'll fix that up - are you okay with the changes otherwise?

Reply:
Yeah, I think that's fine.